src/browser/third-party-grid.ts

```diff
@@ -0,0 +1,152 @@
+// https://github.com/webdriverio/webdriverio/blob/ed92a6f2abfc55b3c4ae887a222371554a7ada61/packages/webdriverio/src/utils/driver.ts
+import type { Capabilities, Options } from "@testplane/wdio-types";
```

Comment:
Why we copy this code from wdio?

Reply:
in order to not import it from wdio.
This sequence:
- "import webdriver"
- "create session using webdriver, while also importing webdriverio"
- "attach to session with webdriverio"

Is faster, than:
- "import webdriverio"
- "create session with webdriverio"

Because in first case we are compiling webdriverio while waiting for session